app/Spago.hs

```diff
@@ -77,16 +77,27 @@ initProject force = do
 
 
 -- | Returns the dir path for a given package
+--   If the package is from a remote git repo, return the .spago folder in which we cloned
```

Comment:
docstring?

Reply:
Haskell docstrings don't have the `|` in multiple lines, [see here](https://www.haskell.org/haddock/doc/html/markup.html)